File: Python/7kyu/vowel_count.py
"""
Created on Thu Jul  27 16:03:02 2017

@author: Dmitry White
"""

# TODO: Return the number (count) of vowels in the given string.
# Consider a, e, i, o, and u as vowels for this Kata.

def getCount(inputStr):
    num_vowels = 0
    vowels = ['a','e','i','o','u']
    for char in inputStr:
        if char in vowels: num_vowels +=1
    return num_vowels

File: Go/7kyu/vowel_count.go
/*
Created on Sat Feb 25 19:58:16 2023
@author: Dmitry White
*/
package kata

import "strings"

/*
	TODO: Return the number (count) of vowels in the given string.
	We will consider 'a, e, i, o, u' as vowels for this Kata (but not 'y').
	The input string will only consist of lower case letters and/or spaces.
*/
const vowels = "aeiou"

func GetCount(str string) (count int) {
	for _, char := range str {
		if strings.Contains(vowels, string(char)) {
			count++
		}
	}
	return count
}
